### PR TITLE
[8.0] Port account move batch validate to 8.0

### PR DESCRIPTION
**Pull request by adrienpeiffer**

### CODE REVIEW

File: account_move_batch_validate/README.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+Account Move Batch Validate
```
Comment (lmignon) [inline account_move_batch_validate/README.rst:4]: Start the file with the licence...

``` rest
.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
    :alt: License
```

**Comment by lmignon** [comment]
:+1:

File: account_move_batch_validate/README.rst (hunk 1 above)
Comment (adrienpeiffer) [inline account_move_batch_validate/README.rst:4]: AGPL-3 Badge added ! Thanks @lmignon

**Comment by lmignon** [comment]
@adrienpeiffer :smile:

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**